ert_shared/storage/database_schema.py

```diff
@@ -118,22 +102,19 @@ class Response(Entity):
     __tablename__ = "response"
     __table_args__ = (
         UniqueConstraint(
-            "realization_id",
             "response_definition_id",
-            name="uq_response_realization_id_reponse_defition_id",
+            "index",
+            name="uq_response_response_definition_id_index",
```

Comment:
`uq_response_definition_id_index`

Reply:
Fixed in `database_schema.py` and migration file.